src/lib/dom.generated.d.ts

```diff
@@ -2343,47 +2354,6 @@ declare var AudioScheduledSourceNode: {
     new(): AudioScheduledSourceNode;
 };
 
-/** A single audio track from one of the HTML media elements, <audio> or <video>.  */
```

Comment:
Curious why these disappeared, they're still part of the standard.

Reply:
Discussion is in https://github.com/microsoft/TSJS-lib-generator/pull/802